R/Run_Mod.R

```diff
@@ -106,8 +131,41 @@ run_mod <- function(data,
 
     # prepare data for modeline
     longdata <- prep_data(dl_sub)
-    priorspec <- prep_priors(max_antigens = longdata$n_antigen_isos,
-                             ...)
+
+    # ---------- CHOOSE MODEL/PRIORS DEPENDING ON `correlated` ----------
+    if (!correlated) {
+      # original (independence) behavior
+      priorspec <- prep_priors(
+        max_antigens = longdata$n_antigen_isos, ...
```

Comment:
https://style.tidyverse.org/syntax.html#long-function-calls:~:text=If%20a%20function%20call%20is%20too%20long%20to%20fit%20on%20a%20single%20line%2C%20use%20one%20line%20each%20for%20the%20function%20name%2C%20each%20argument%2C%20and%20the%20closing%20)

```suggestion
        max_antigens = longdata$n_antigen_isos, 
        ...
```